frontend/packages/vkt/src/components/publicEnrollment/PublicEnrollmentPaymentSum.tsx

```diff
@@ -0,0 +1,30 @@
+import { H1 } from 'shared/components';
+
+import { usePublicTranslation } from 'configs/i18n';
+import { PublicEnrollmentFormStep } from 'enums/publicEnrollment';
+import { PublicEnrollment } from 'interfaces/publicEnrollment';
+import { PublicEnrollmentUtils } from 'utils/publicEnrollment';
+
+export const PublicEnrollmentPaymentSum = ({
+  activeStep,
+  enrollment,
+}: {
+  activeStep: PublicEnrollmentFormStep;
+  enrollment: PublicEnrollment;
+}) => {
+  const { t } = usePublicTranslation({
+    keyPrefix: 'vkt.component.publicEnrollment.paymentSum',
+  });
+
+  const sum = PublicEnrollmentUtils.calculateExaminationPaymentSum(enrollment);
+
+  const content = `${t('title')}: ${sum.toFixed(2).replace('.', ',')} €`;
```

Comment:
Tää näyttäis olevan parissa paikassa niin tosta vois tehä ehkä oman utilin et muuttaa numeerisen arvon € muotoiseen esitystapaan